codecarbon/core/cpu.py

```diff
@@ -0,0 +1,99 @@
+"""
+Implements tracking Intel CPU Power Consumption on Mac and Windows
+using Intel Power Gadget https://software.intel.com/content/www/us/en/develop/articles/intel-power-gadget.html
+"""
+import os
+import shutil
+import subprocess
+import sys
+from logging import getLogger
+from typing import Dict
+
+import pandas as pd
+
+logger = getLogger(__name__)
+
+
+def is_powergadget_available():
+    try:
+        IntelPowerGadget()
+        return True
+    except Exception as e:
+        logger.debug(
+            f"Exception occurred while instantiating IntelPowerGadget : {e}",
+            exc_info=True,
+        )
+        print(e)
```

Comment:
Let's not print the exception, the logging should do. We do not want the linux users to get this every time..
```suggestion
```